### PR TITLE
AHOYAPPS-317: Update issue templates

### PR DESCRIPTION
**Pull request by timrozum**
https://issues.corp.twilio.com/browse/AHOYAPPS-317

#### Changes

1. Used a combo of new default templates from GitHub and quickstart example.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+> #### Before Creating an Issue
+>
+> - If there is an issue specific to the video app report it here. If there is an issue or question related to the  video SDK please [create an SDK issue](https://github.com/twilio/twilio-video-ios/issues) instead.
```
Comment (paynerc) [inline .github/ISSUE_TEMPLATE/bug_report.md]: `the  video SDK` => extra space

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+> - Make sure the issue is not already addressed by [docs](https://www.twilio.com/docs/video) or [existing issues](https://github.com/twilio/twilio-video-app-android/issues).
```
Comment (paynerc) [inline .github/ISSUE_TEMPLATE/bug_report.md]: `https://github.com/twilio/twilio-video-app-android/issues` => says Android. 😀

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+> #### Before Creating an Issue
+>
+> - If there is an issue specific to the video app report it here. If there is an issue or question related to the  video SDK please [create an SDK issue](https://github.com/twilio/twilio-video-ios/issues) instead.
+> - Make sure the issue is not already addressed by [docs](https://www.twilio.com/docs/video) or [existing issues](https://github.com/twilio/twilio-video-app-android/issues).
```
Comment (paynerc) [inline .github/ISSUE_TEMPLATE/feature_request.md]: https://github.com/twilio/twilio-video-app-android/issues

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (timrozum) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (timrozum) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Oh man caught copying from John.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (timrozum) [reply .github/ISSUE_TEMPLATE/bug_report.md]: ✅

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (timrozum) [reply .github/ISSUE_TEMPLATE/bug_report.md]: ✅

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (timrozum) [reply .github/ISSUE_TEMPLATE/feature_request.md]: ✅

**Pull request merged**